pilz_teleoperation/src/pilz_teleoperation/ros_message_serializer.py

```diff
@@ -0,0 +1,53 @@
+import genpy
+import rospy
+
+class RosMessageSerializer:
```

Comment:
allows for better inheritance
```suggestion
class RosMessageSerializer(object):
```